docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+---
+sidebar_position: 2
+title: Tutorial
+tags: [quickstart]
+---
+
+:::info
+
+This tutorial doesn’t assume any existing knowledge of the blindnet devkit or even any privacy-related technologies.
+
+:::
+
+The purpose of the following tutorial is to serve as an entry point for the blindnet devkit, by simply walking you through its different components to give you a better understanding of the overall structure and logic.
+
+Doing so, we'll answer the following questions:
+
+- What is the purpose of the Privacy Computation Engine;
```

Comment:
For all the linkable documentation concepts (like here Privacy Computation Engine), I would suggest to adopt the same rule as on wikipedia. The first time the concept is mentioned in a text (or a section of a text if it is expected that sections might be read independently), the concept is linked to the corresponding documentation page. 

Every next time, it is just noted in the First Letter Upper Case notation without beining linked.